ui/ceval/package.json

```diff
@@ -14,21 +14,22 @@
   ],
   "author": "Thibault Duplessis",
   "license": "AGPL-3.0-or-later",
-  "scripts": {
-    "compile": "tsc --incremental",
```

Comment:
any reason to remove the `--incremental`?

Reply:
The scripts aren't very useful yet, they're placeholders.  It would be cool if they could drive esbuild via command line and bypass bleep entirely, but that's a long-term goal.

You can always specify --incremental on the command line if you'd like.  Bleep itself always does a --force build when not in watch mode, and watch builds are of course incremental.